inc/saiport.h

```diff
@@ -136,6 +136,9 @@ typedef enum _sai_port_fdb_learning_mode_t
     /** Trap packets with unknown source MAC to CPU. Do not learn. Forward based on destination MAC */
     SAI_PORT_FDB_LEARNING_MODE_CPU_LOG,
 
+    /** Send MAC notification via FDB callback. Do not learn. Do not forward */
+    SAI_PORT_FDB_LEARNING_MODE_SW_TRAP,
```

Comment:
@lguohan, for this issue to be sorted, one needs to have a FDB entry added as "pending". So on the first packet add that entry to the HW and then let the stack do verification etc and then make the pending entry active.